test/esm.mjs

```diff
@@ -0,0 +1,10 @@
+import assert from 'assert'
+import StyleToObject from '../index.mjs'
+
+describe('it works with ESM', () => {
+	it('imports the functiion without errors', () => {
```

Comment:
Nit: we prefer 2 spaces over 1 tab

Reply:
@remarkablemark oops sorry will fix

btw i recommend setting up git hooks to run formatter and linter before committing